src/sources/audio_base_latency.ts

```diff
@@ -1,4 +1,12 @@
+import { isAndroid, isChromium } from '../utils/browser'
+
 export default function getAudioContextBaseLatency(): number | undefined {
+  // The signal emits warning in console, so is is enabled only on Android where it's less visible
+  const isAllowedPlatform = isAndroid() && isChromium()
```

Comment:
> Let's run in all Android browsers, including Firefox, because it's hard to see the browser console in all of them.

I can skip Chromium check although in firefox it looks useless because it always returns `0`

> Let's also enable it on WebKit because it doesn't print the warning (please double-check).

It seems so, I'll enable for webkit. Although it may be risky, they can enable the warning same as in blink and gecko

Reply:
> I can skip Chromium check although in firefox it looks useless because it always returns `0`

The code will become simpler, so why not

> It seems so, I'll enable for webkit. Although it may be risky, they can enable the warning same as in blink and gecko

No problem to disable when they enable the warning